src/vcFBX.cpp

```diff
@@ -275,7 +275,7 @@ void vcFBX_CleanMaterials(udChunkedArray<vcFBXMaterial> *pMats)
         void *pPtr = (*pMats)[i].textures[j].pPixels;
         for (uint32_t k = i; k < pMats->length; ++k)
         {
-          for (uint32_t l = j + 1; l < (*pMats)[k].textures.length; ++l)
+          for (uint32_t l = j; l < (*pMats)[k].textures.length; ++l)
```

Comment:
To confirm, setting `(*pMats)[i].textures[j].pPixels` to `nullptr` after assigning `pPtr` doesn't work?

Reply:
Yeah, I'm as shocked as you, tried like twice